src/nvim/usercmd.c

```diff
@@ -1431,7 +1431,6 @@ static size_t uc_check_code(char *code, size_t len, char *buf, ucmd_T *cmd, exar
 
   l++;
   if (l <= 1) {
-    type = ct_NONE;
```

Comment:
Hmm, the code formatted by uncrustify look a bit ugly. Maybe just comment out this line instead of removing.

Reply:
I was thinking something like deleting it outright wouldn't look so good, and lint seems to fail on this because of the deletion.